.circleci/config.yml

```diff
@@ -88,9 +109,7 @@ workflows:
                   - master
       - publish-github-release:
           context: general-vars
-          requires:
```

Comment:
Please do not delete this as release on GH should be created after successfully publishing to npmjs